pkg/services/featuremgmt/registry.go

```diff
@@ -1362,6 +1362,12 @@ var (
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
 		},
+		{
+			Name:        "alwaysSendUsageInisghts",
+			Description: "Forces sending usage insights data no matter what if config.analytics.enabled is enabled",
+			Stage:       FeatureStageExperimental,
+			Owner:       grafanaSharingSquad,
```

Comment:
should that be Enterprise instead?

Reply:
They don't have team in GH, and they don't appears in owners file